package.json

```diff
@@ -126,15 +130,39 @@
                         ".",
                         "..\\fracas\\lib"
                     ],
-                    "markdownDescription": "A list of paths to search for Racket collections.",
+                    "markdownDescription": "Paths to search for Racket collections. Relative to `Project Dir`",
                     "order": 3
                 },
                 "vscode-fracas.general.ninjaPath": {
                     "type": "string",
                     "default": "ninja",
-                    "markdownDescription": "Specifies the path to the `ninja` executable used to precompile racket files.",
+                    "markdownDescription": "Path to the `ninja` executable used to precompile racket files.",
                     "order": 4
                 },
+                "vscode-fracas.localization.stringTableRegistryFile": {
+                    "type": "string",
+                    "default": "..\\tdp1.unreal\\Source\\Data\\TdpLocalization.cpp",
+                    "markdownDescription": "Path to the generated C++ source file that registers localized text tables with the UE4 project. Relative to *Project Dir*",
+                    "order": 1
+                },
+                "vscode-fracas.localization.stringTableSourcePaths": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    },
+                    "default": [
+                        "fracas\\localization-defines.frc",
+                        "fracas\\text\\localized-text*.frc"
+                    ],
+                    "markdownDescription": "Glob patterns of fracas source files containing localized text. Used to generated the *String Table Registry File*. Relative to `Project Dir`",
+                    "order": 2
+                },
+                "vscode-fracas.localization.stringTableDestDir": {
+                    "type": "string",
+                    "default": "..\\tdp1.unreal\\Content\\GameData\\Localization\\",
+                    "markdownDescription": "Directory with generated CSV files containing localized text tables. Relative to *Project Dir*",
+                    "order": 3
```

Comment:
These entries define new settings for localized text output
![image](https://user-images.githubusercontent.com/5162625/154780591-9d00509c-416c-4725-a175-9b5820def9c8.png)